mmv1/third_party/terraform/tests/resource_spanner_database_test.go.erb

```diff
@@ -69,6 +75,33 @@ func TestAccSpannerDatabase_basic(t *testing.T) {
 				ImportStateVerify:       true,
 				ImportStateVerifyIgnore: []string{"ddl", "deletion_protection"},
 			},
+			{
```

Comment:
note: What you're doing here is importing the resource over and over again with different import formats. You aren't running `terraform apply`, for example. Is that what you intended?